x-pack/plugins/ml/common/constants/search.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const DEFAULT_QUERY_SIZE = 500;
```

Comment:
💯

Reply:
I agree there might be different query sizes for different purposes. Do you think it would still make sense to collect them in this file, for example:

```javascript
export const ANNOTATIONS_DEFAULT_QUERY_SIZE = 500;
export const ANOMALIES_DEFAULT_QUERY_SIZE = 500;
```

I think I would prefer that instead of spreading it across files. Let me know what you think.

Reply:
Yes, makes sense to have them defined in a single place in this file. Maybe `ANNOTATIONS_TABLE_DEFAULT_QUERY_SIZE` and `ANOMALIES_TABLE_DEFAULT_QUERY_SIZE` might be clearer variable names?

Reply:
Cool, done in c5a4a11c63.